src/main/java/com/khomishchak/cryproportfolio/controllers/ExchangerController.java

```diff
@@ -40,4 +41,10 @@ public Balance getAccountBalance(@PathVariable long accountId, @PathVariable Str
     public List<Balance> getAccountBalances(@PathVariable long accountId) {
         return exchangerService.getAllMainBalances(accountId);
     }
+
+    @GetMapping("/wallet/{accountId}/get/deposit-withdrawal-history/{exchangerCode}")
```

Comment:
Do not use any very such as `get`,`create`,`update` in the REST. You already have marked this endpoint as `GET`
```suggestion
    @GetMapping("/wallet/{accountId}/deposit-withdrawal-history/{exchangerCode}")
```